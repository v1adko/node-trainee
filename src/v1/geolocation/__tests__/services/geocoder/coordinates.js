import { geocoder } from '../../../';

jest.setTimeout(10000);

describe('Test the receiving address from coordinates', () => {
  it('should return address', async () => {
    const address = await geocoder.coordinatesToAddress(50, 30);
    expect(address).toMatchSnapshot();
  });

  it('should reject if no exist matched of coordinates', async () => {
    const error = await geocoder.coordinatesToAddress(444, 444);
    expect(error).toMatchSnapshot();
  });
});
