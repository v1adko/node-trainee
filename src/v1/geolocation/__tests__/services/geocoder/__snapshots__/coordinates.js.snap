// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the receiving address from coordinates should reject if no exist matched of coordinates 1`] = `
Object {
  "error": "Error",
  "message": "Response status code is 400",
}
`;

exports[`Test the receiving address from coordinates should return address 1`] = `
Array [
  Object {
    "address": "Unnamed Road, Kyivs'ka oblast, Ukraine",
    "coordinates": Object {
      "lat": 49.999137,
      "lon": 30.0019538,
    },
  },
]
`;
